test/cql-pytest/test_describe.py

```diff
@@ -447,6 +447,24 @@ def test_desc_index(cql, test_keyspace):
         
         assert f"CREATE INDEX named_index ON {tbl}{maybe_space}(c)" in c_desc
 
+def test_desc_index_on_collections(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, "a int primary key, b frozen<set<int>>, c map<int, int>, d list<int>, e set<int>") as tbl:
+        indexes = [
+            ("idx_b",           f"CREATE INDEX idx_b ON {tbl}(full(b))"),
+            ("idx_c_entries",   f"CREATE INDEX idx_c_entries ON {tbl}(entries(c))"),
+            ("idx_c_keys",      f"CREATE INDEX idx_c_keys ON {tbl}(keys(c))"),
+            ("idx_c_values",    f"CREATE INDEX idx_c_values ON {tbl}(values(c))"),
+            ("idx_d",           f"CREATE INDEX idx_d ON {tbl}(d)"),    # TODO: this index will fail the assert becuase its target is "values(d)"
```

Comment:
Yes, for list, indexing `d` and `values(d)` is exactly the same thing (the target will always be written as `values(d)` and that's what you can expect.